server/data/localMockData/actionPlanResponse.ts

```diff
@@ -2,26 +2,40 @@ import { ActionPlanResponse } from '../../interfaces/educationAndWorkPlanApi/act
 import { GoalResponse } from '../../interfaces/educationAndWorkPlanApi/goalResponse'
 import { StepResponse } from '../../interfaces/educationAndWorkPlanApi/stepResponse'
 
-const aValidActionPlanResponseWithOneGoal = (): ActionPlanResponse => {
+const aValidActionPlanResponse = (options?: {
+  reference?: string
```

Comment:
I love an options object on our test data builders - it makes them so flexible in their use 😁 (see examples of their use in the tests below)